plugins/woocommerce-admin/client/marketplace/components/content/content.tsx

```diff
@@ -185,7 +186,12 @@ export default function Content(): JSX.Element {
 		<div className="woocommerce-marketplace__content">
 			<Promotions />
 			<InstallNewProductModal products={ products } />
-			<ConnectNotice />
+			{ selectedTab !== 'business-services' && (
+				<>
+					<ConnectNotice />
+					<PluginInstallNotice />
+				</>
+			) }
```

Comment:
If we wanted to remove `<ConnectNotice />` from My Subscriptions screen (there's already information about missing connection there), the code that I tested is:
 
```suggestion
			{ selectedTab !== 'business-services' && <PluginInstallNotice /> }
			{ selectedTab !== 'business-services' &&
				selectedTab !== 'my-subscriptions' && <ConnectNotice /> }
```

TBD though if we want to change it.

Reply:
Done in e01b89a after getting confirmation from @hesahesa